plugins/inventory/lxd.py

```diff
@@ -49,26 +50,38 @@
             - If I(trust_password) is set, this module send a request for authentication before sending any requests.
             type: str
         state:
-            description: Filter the container according to the current status.
+            description: Filter the instance according to the current status.
             type: str
             default: none
             choices: [ 'STOPPED', 'STARTING', 'RUNNING', 'none' ]
-        prefered_container_network_interface:
+        typefilter:
```

Comment:
```suggestion
        type_filter:
```
How about renaming this option to `type_filter`? This fits better to the other options IMO.